### ISSUE TITLE
We cannot pass checkpoint and add account which was added

### ISSUE THREAD

**Issue by microdesign** [thepostcode/nextpost#61]
We cannot pass checkpoint and add account which was added - no errors, nothing?

**Comment by sameeraaa** [comment]
What's the actual problem you're facing? Kindly write in details.

**Comment by microdesign** [comment]
1. We were having an account added and worked in your system
2. The system told it must be revalidated
3. I enter my login and see the test to go to Instagram to pass the checkpoint
4. I click everything seem to work 
5. Profile is not added

???

**Comment by sameeraaa** [comment]
@microdesign 

- Which version of nextpost you're using?
- Did you use private proxy?

**Comment by microdesign** [comment]
Last version.

No I dont use now and before when I was able to add my Instagram account

**Comment by andrejcook** [comment]
I have the exact same problem. My accounts used to work fine for about 5 days since I just started using the product.  But then noticed my "post or schedule" button greyed out all of a sudden unable to log back into my account via nextpost.  Continues to fail even when I validate my IG account with the code received. No luck.  

Unable to use the app at all right now. For fun I tried just logging in a few times on instagram.com and IG validates fine via browser and IG mobile app. No blocks, but NextPost won't allow me to log in to my account. Why now?  Have you made changes the app?  Also tried using a VPN and same issue.   I tried 3 different IG accounts. They all have the same problems.

I have a campaign running and need this resolved asap, please. That was the whole point of buying the license and going through the long process of setting it up.

![image](https://user-images.githubusercontent.com/34035953/44536292-5761fc00-a6b9-11e8-8f24-d7ccc0679ba5.png)

**Comment by sameeraaa** [comment]
@microdesign  @andrejcook  Looks like your server IP is listed as spam by IG, Try to use Private proxies.

**Comment by microdesign** [comment]
@sameeraaa Let you and your team find another excuse because with many tries and debugging in the code we was able to login back our account ?!??!

**Comment by sameeraaa** [comment]
@microdesign I'm not the developer but I'm able to add accounts for me & my clients. I recommend using Luminati Datacenter Proxies https://goo.gl/cYy9WK

**Comment by sameeraaa** [comment]
@microdesign The current version of nextpost v4.2.0 has native login flow means it's currently the best solution available!

**Comment by microdesign** [comment]
It is not possible one time to be a spam and one time to not be. 
Also, you dont say your customers in CodeCanyon they must pay for another service to use your software.

I`m with the last version !

**Comment by sameeraaa** [comment]
@microdesign We don't know when IG will spam your IP, If you're unable to add account even in latest version it means your server IP is listed as Spam and IG throttle your Request!

**Comment by andrejcook** [comment]
I have only shared 5 posts in the last week. Span has nothing to do with it. ;)  I'm mindful of how I use these type of products. My IP cant blocked I can still post using the Windows app or browser. There is a different issue.

**Comment by sameeraaa** [comment]
@andrejcook You're using Nextpost on your local server?

**Comment by andrejcook** [comment]
Yes correct. Just started using it 5 days ago. Great UI and product. No complaints others than mentioned issue

**Comment by sameeraaa** [comment]
@andrejcook Just give a try to proxy and check if it works.

**Comment by sameeraaa** [comment]
Working fine for me!

**Comment by andrejcook** [comment]
Let me try again. Otherwise, I will try your proxy recommendation.

**Comment by sameeraaa** [comment]
@andrejcook Okay great

**Comment by microdesign** [comment]
@sameeraaa so they listed me as SPAM and after debugging from our side they UN-SPAM our server :)

**Comment by andrejcook** [comment]
Yeah still no luck. Even after verifying my account with IG 6 digit code received, it fails me from logging in.  Where can I check my version number?   Need to use proxy

MD. can you explain the un-spam process, please. How did you debug for IG to remove you fro spam IP?

**Comment by sameeraaa** [comment]
@microdesign Which version you're currently using?

**Comment by sameeraaa** [comment]
@andrejcook There is no way to un-spam/un-block by IG server.

**Comment by andrejcook** [comment]
@sameeraaa I have an existing account with Express VPN. Do you know if they offer a similar proxy solution than luminati

**Comment by sameeraaa** [comment]
@andrejcook  No Express VPN didn't offer proxies, But Luminati has it's own VPN service also!

**Comment by sameeraaa** [comment]
@andrejcook  The proxies you buy on luminati can also be used via there chrome extension & android/iOS app.

**Comment by andrejcook** [comment]
OK Thanks

**Comment by microdesign** [comment]
@sameeraaa I know but if you cannot help me - why you answer me here?

**Comment by sameeraaa** [comment]
@microdesign Okay, wait here for developer response :)

**Comment by microdesign** [comment]
@sameeraaa  you are right maybe I must and put one start in codecanyon and then someone will going to answer me

**Comment by sameeraaa** [comment]
@microdesign also note that Postcode (The Developer) is on holidays and will be available after 1st Sep!

**Comment by andrejcook** [comment]
Thanks @microdesign You mentioned 4.2 is the latest version which I believe I have since CodeCantyon has not sent me any new emails. But where can I check the v number in he app or code just to make sure? Thanks. Do you know?

**Comment by sameeraaa** [comment]
@andrejcook Check in app/config/common.config.php

**Comment by andrejcook** [comment]
Ahhh great . I need an upgrade after all
define("APP_VERSION", "4.1.1"); 
Thank you.

**Comment by sameeraaa** [comment]
@andrejcook Try adding account in latest version and give feedback.

**Comment by andrejcook** [comment]
Good news. My previously blocked IG account is now reconnecting again after the 4.2 upgrade! 
But for the life of me, I can not figure out how to add a proxy.  According to the app "Proxy should match the following pattern: http://ip:port OR http://username:password@ip:port"  

I tried this and it's not accepting new connections. All I can think is that Luminati needs additional steps which in itself is NOT an easy product to understand for the average social media manager.

It would be great if someone can share a sample link here of how they got Luminati proxy service to work in NextPost please. Thanks

**Comment by sameeraaa** [comment]
@andrejcook Send me mail at sameerakhan1218@gmail.com, I'll help you in setting up luminati proxy

**Comment by microdesign** [comment]
Same problem the account was logged out and now we must switch to developing to add I dont want to use this system like this, please make it work or going to ask refund of the system and the modules I bought.

Also, it is very easy to be hacked my server has I files from third party from your script?!?!?!
